packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -115,6 +116,14 @@ export interface IDetailsListProps extends IBaseProps<IDetailsList>, IWithViewpo
    */
   isHeaderVisible?: boolean;
 
+  /**
+   * Controls the visibility of the details footer.
+   * @default false
+   */
+  isFooterVisible?: boolean;
```

Comment:
These 3 new props seem somewhat redundant to me. It almost feels like just the onRenderDetailsFooter optional callback should be enough.
If no value is provided for the callback, footer is not visible.
Also, footerText is just one of the many ways we can render a footer. We should be able to capture it in IDetailsFooterProps.

Reply:
@aditima, please review/approve the changes